enterprise/frontend/src/embedding-sdk/hooks/public/use-metabase-auth-status.ts

```diff
@@ -1,4 +1,19 @@
+import { useContext } from "react";
+import { ReactReduxContext } from "react-redux";
+import { t } from "ttag";
+
 import { useSdkSelector } from "embedding-sdk/store";
 import { getLoginStatus } from "embedding-sdk/store/selectors";
 
-export const useMetabaseAuthStatus = () => useSdkSelector(getLoginStatus);
+// eslint-disable-next-line no-literal-metabase-strings -- this string only shows in the console.
+export const USE_OUTSIDE_OF_CONTEXT_MESSAGE = t`The useMetabaseAuthStatus hook must be used within a component wrapped by the MetabaseProvider`;
```

Comment:
I don't think we want to localize error messages/logs. It was done in the BE and it was removed because it made debugging customer issues harder

Reply:
I see! I'll get rid of that then 👍🏻